src/components/SharedComponents/Buttons/AddObsButton.js

```diff
@@ -20,19 +37,30 @@ const AddObsButton = ( ): React.Node => {
         closeModal={closeModal}
         modal={<AddObsModal closeModal={closeModal} />}
       />
-      <IconButton
-        icon="plus-sign"
+      <Pressable
         onPress={openModal}
-        size={40}
-        mode="contained"
-        containerColor={theme.colors.secondary}
-        iconColor={theme.colors.onSecondary}
+        testID="add-obs-button"
+        disabled={false}
         accessibilityLabel={t( "Observe" )}
         accessibilityHint={t( "Opens-add-observation-modal" )}
-        className="m-0"
-        disabled={false}
-        testID="add-obs-button"
-      />
+        accessibilityRole="button"
+        accessibilityState={{
+          disabled: false
+        }}
+        style={shadow}
+      >
+        <View className="w-[69px] h-[69px] rounded-full overflow-hidden">
```

Comment:
Just curious, is there a need for this View here?

Reply:
I think it could be omitted if `Pressable` is wrapped as a styled component so it can consume the `className` prop.